content/calloop-v0.7.markdown

```diff
@@ -0,0 +1,121 @@
+Title: Version 0.7 of calloop
+Date: 2020-10-09 19:00
+Category: Releases
+Slug: calloop-v-0-7
+Authors: Victor Berger
+Summary: Announcement of version 0.7 of calloop, and a presentation of the crate.
+
+We just released [version 0.7](https://github.com/Smithay/calloop/releases/tag/v0.7.0) of
+[calloop](https://docs.rs/calloop), the callback-based event loop that backs
+[Smithay](https://github.com/Smithay/smithay) and optionnaly
+[SCTK](https://github.com/Smithay/client-toolkit). As the crate is now getting mature, it is time
```

Comment:
Reading the markdown form would be easier if the links were just `[SCTK]` with the target defined at the end of the section as

```md
[SCTK]: https://github.com/Smithay/client-toolkit
```

;)